app/javascript/packs/admin_users/user.jsx

```diff
@@ -48,6 +49,7 @@ class User extends React.Component {
     this.checkCustomSkillButton = this.checkCustomSkillButton.bind(this);
     this.addCustomSkill = this.addCustomSkill.bind(this);
     this.findSkillTitleById = this.findSkillTitleById.bind(this);
+    this.saveChanges = this.saveChanges.bind(this)
```

Comment:
`;`. Can you install a `Linter` for your IDE? If there is no appropriate linter I do recommend to use another editor for `javascript`